deps/github.com/openshift/kubernetes/openshift-hack/rebase.sh

```
@@ -125,51 +124,64 @@ fi
 go_mod_go_ver=$(grep -E 'go 1\.[1-9][0-9]?' go.mod | sed -E 's/go (1\.[1-9][0-9]?)/\1/' | cut -d '.' -f 1,2) # Need to handle mod versions like 1.23 and 1.23.4; our release images only have major.minor
-tag="rhel-8-release-golang-${go_mod_go_ver}-openshift-${openshift_release#release-}"
-
-# update openshift go.mod dependencies
-sed -i -E "/=>/! s/(\tgithub.com\/openshift\/[a-z|-]+) (.*)$/\1 $openshift_release/" go.mod
+tag=$(grep "^  tag:" .ci-operator.yaml | head -n1 | sed -E 's/.*: (.*)/\1/')
 
 echo "> go mod tidy && hack/update-vendor.sh"
-podman run -it --rm -v "$(pwd):/go/k8s.io/kubernetes:Z" \
+podman run --rm -v "$(pwd):/go/k8s.io/kubernetes:Z" \
   --workdir=/go/k8s.io/kubernetes \
   "registry.ci.openshift.org/openshift/release:$tag" \
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)